src/api/mapping.ts

```diff
@@ -15,7 +20,40 @@ mappingRouter.post('/', (req, res) => {
   }
   domainKeys.push(mappingObject)
   setData('mappings', domainKeys)
-  res.json(mappingObject)
+  const projectFolder = `${req.body.domain}`
+  const prodConfig = {
+    apps: [
+      {
+        name: 'myProxy-prod',
+        script: './build/app.js',
+        instances: 1,
+        autorestart: true,
+        watch: false,
+        max_memory_restart: '1G',
+        env_production: {
+          NODE_ENV: 'production',
+          PORT: parseInt(req.body.port, 10),
+          ADMIN: '123'
+        }
+      }
+    ]
+  }
+  exec(`
+    mkdir -p ~/projects
+    mkdir ~/projects/${projectFolder}
+    git init ~/projects/${projectFolder}
+    cp ${path.join(
+    __dirname,
+    '../../scripts'
+  )}/post-receive ~/projects/${projectFolder}/.git/hooks/
+    cd ~/projects/${projectFolder}
+    echo "module.exports =" > deploy.config.js
+    echo '${JSON.stringify(prodConfig)}' >> deploy.config.js
```

Comment:
could you combine this into 1 command? 
`echo 'module.exports = JSON.stringify(prodConfig)'`